src/app/src/main/kotlin/org/imperial/mrc/hint/AppProperties.kt

```diff
@@ -7,6 +7,12 @@ import java.net.URL
 import java.util.*
 
 interface AppProperties {
+    val adrANC: String
+    val adrART: String
+    val adrPJNZ: String
+    val adrPop: String
+    val adrShape: String
+    val adrSurvey: String
```

Comment:
Since all of these are adrSchema, could we maybe rename adrSchema to make it clear what that is? (It's the dataset schema??)